### PR TITLE
Initial portfolio setup

### PR DESCRIPTION
**Pull request by Danie12345**
The first major advance in the portfolio.
Includes:

Header, Headline and Background for mobile.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Daniel's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body class="body-container">
+    <header class="header">
+        <span class="navitem name">Daniel Malo</span>
```
Comment (smunozmo) [inline index.html]: - [x] Please use an `a` tag for your logo, it should be clickable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+        </section>
+        <div class="media-icons">
+            <img class="media-icon" src="Icons/Vector.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-1.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-2.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-3.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-4.png" alt="no image available">
+        </div>
+    </div>
```
Comment (smunozmo) [inline index.html]: - [x]  Nice work here, it would be better if you used a `section` tag instead of a `div` for this section.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <span>Hey There.</span>
+            <span>I'm Daniel</span>
+        </h1>
+        <section>
+            <h3>I'm a Software Developer</h3>
+            <div class="self-intro">
+                I can help you build a product, feature or website. Look through some of 
+                my work and experience it! If you like what you see and have a project 
+                you need coded, please don't hesitate to contact me.
+            </div>
```
Comment (smunozmo) [inline index.html]: - [x] This is looking good, please use a `p` tag for this text.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        </section>
+        <div class="media-icons">
+            <img class="media-icon" src="Icons/Vector.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-1.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-2.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-3.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-4.png" alt="no image available">
+        </div>
```
Comment (smunozmo) [inline index.html]: - [x] You are almost there, however, it's important to use an `ul` tag here and every icon should be listed with `li` tags and `a` tags to make them clickable, like this:

```
<ul>
    <li><a>content</a></li>
    <li><a>content</a></li>
    <li><a>content</a></li>
</ul>
````

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Danie12345, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fixing your linters problem, try to set it up again to avoid a project invalidation in the following reviews. 

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Danie12345) [reply index.html]: @smunozmo Sorry, I do not understand. My name is supposed to also be my logo?

Comment (Danie12345) [reply index.html]: I have considered it so, and changed it

**Review by CollinsTatang — APPROVED** [review]
Hi @Danie12345, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

### **OPTIONAL CHANGES**

- [ ] Please do corrections on your design the background image is overlapping the text and the header cover the heading text.
**Take a look**
overlap
**Figmal Design**
figma


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**